### PR TITLE
Adds deliciously revolting bone-breaking sounds.

### PR DESCRIPTION
**Pull request by FreeStylaLT**
Adds 6 new sound files to play when a bone is broken:

Video demonstration:
https://my.mixtape.moe/bzuzur.mp4

🆑 FreeStylaLT
add: Crunchy sounds when bones are broken
/🆑

### CODE REVIEW

**Comment by Spartan6** [comment]
Sounds like Kidan break dancing.

**Comment by KasparoVy** [comment]
Very nice.

**Comment by Tails2091** [comment]
This is a nice touch but I noticed some things, 1-3 are mono, the rest are stereo, all files are stereo files. 4 and 6 are the same sound but panned differently, 5 is panned completely right.
Bonus: all levels in mono;
levels

File: code/modules/surgery/organs/organ_external.dm [hunk 1]
```diff
@@ -607,6 +607,8 @@ Note that amputating the affected organ does in fact remove the infection from t
 			"<span class='warning'>You hear a loud cracking sound coming from \the [owner].</span>",\
 			"<span class='danger'>Something feels like it shattered in your [name]!</span>",\
 			"You hear a sickening crack.")
+		var/S = "sound/effects/bone_break_[rand(1,6)].ogg"
```
Comment (Fox-McCloud) [inline code/modules/surgery/organs/organ_external.dm]: Just put this whole thing in `playsound`. Also, spacing.

**Comment by variableundefined** [comment]
@FreeStylaLT

File: code/modules/surgery/organs/organ_external.dm [hunk 2]
```diff
@@ -607,8 +607,7 @@ Note that amputating the affected organ does in fact remove the infection from t
 			"<span class='warning'>You hear a loud cracking sound coming from \the [owner].</span>",\
 			"<span class='danger'>Something feels like it shattered in your [name]!</span>",\
 			"You hear a sickening crack.")
-		var/S = "sound/effects/bone_break_[rand(1,6)].ogg"
-		playsound(owner,S, 150, 1)
+		playsound(owner,"sound/effects/bone_break_[rand(1,6)].ogg", 150, 1)
```
Comment (KasparoVy) [inline code/modules/surgery/organs/organ_external.dm]: Spacing after `owner,`
Might as well be consistent, sorry for the nitpick

**Comment by KasparoVy** [comment]
God bless haha

**Comment by variableundefined** [comment]
@FreeStylaLT When can you address Tiger's request?

**Pull request merged**